.gitignore

```diff
@@ -60,3 +60,52 @@ bin/
 *.class
 build/
 .gradle/
+
```

Comment:
yea, its my mistake, i will remove gitignore file and will make new pull request.